CONTRIBUTING.md

```diff
@@ -376,6 +376,25 @@ therefore helping yourself and other engineers.
 
 You can find the [enums of event values here]()
 
+### Auditing
+
+For those operations requiring an audit trial, we use the [CDP Auditing library](https://github.com/DEFRA/cdp-libraries/tree/main/packages/cdp-auditing#readme).
+
+The library provides a simple `audit` function accepts either a string or a serialisable object.
+
+```js
+import { audit } from '@defra/cdp-auditing'
+audit('This is a test of auditing')
+// {"log.level":"audit","time":"2025-07-04T11:46:33.278Z","msg":"This is a test of auditing"}
+
+audit({ id: '1234', event: 'login', outcome: 'denied' }, 'Login failed')
```

Comment:
I wonder if these examples should follow our implementation in code a little more closely and I imagine that we'll only be using the serialisable object approach

Reply:
I've made those changes.